templates/terraform/modules/s3_hosting/main.tf

```diff
@@ -0,0 +1,103 @@
+resource "aws_s3_bucket" "www" {
+  // Our bucket's name is going to be the same as our site's domain name.
+  bucket = "${var.bucket_name}"
+  // Because we want our site to be available on the internet, we set this so
+  // anyone can read this bucket.
+  acl    = "public-read"
```

Comment:
At some point this should be changed to private, with proper access set up from Cloudfront, we have plenty of examples of that. Not important for now though.

Reply:
yup true, I just wanted to be a bit easier for demoing right now. We can just use the s3 url to load it.